File: TOKILearning/v1/1D/toki1D3.pas
program swap;

var
    a, b: integer;

procedure swap(var x,y: integer);
var
    temp: integer;
begin
    temp := x;
    x := y;
    y := temp;
end;

begin
    readln(a, b);
    swap(a, b);
    writeln(a, ' ', b);
end.

File: TOKILearning/v1/1D/toki1d3.cpp
#include <iostream>
using namespace std;

void swap(int &a,int &b){
	int temp;
	temp=a;
	a=b;
	b=temp;
}

int main(){
	int m,n;
	cin>>m>>n;
	swap(m,n);
	cout<<m<<" "<<n<<"\n";
	return 0;
}
